File: problemas/strings/2157/2157.dart
import 'dart:io';

void main() {
  int n = int.parse(stdin.readLineSync()!);
  for (int i = 0; i < n; ++i) {
    List<int> entrada = stdin.readLineSync()!.split(' ').map((x) => int.parse(x)).toList();
    int b = entrada[0];
    int e = entrada[1];

    String resposta = "";
    for (int j = b; j <= e; ++j) {
      resposta += j.toString();
    }
    resposta += resposta.split('').reversed.join();
    
    print(resposta);
  }
}

File: problemas/strings/2157/2157.swift
import Foundation

let n = Int(readLine(strippingNewline: true)!)!
for _ in 0..<n {
    let entrada = readLine(strippingNewline: true)!.components(separatedBy: " ")
    let b = Int(entrada[0])!
    let e = Int(entrada[1])!

    var resposta = ""
    for i in b...e {
        resposta += String(i)
    }
    resposta += resposta.reversed()

    print(resposta)
}